shell/src/main/java/alluxio/cli/docgen/MetricsDocGenerator.java

```diff
@@ -110,6 +120,39 @@ public static void generate() throws IOException {
             key, StringEscapeUtils.escapeHtml4(metricKey.getDescription().replace("'", "''"))));
       }
     }
+    if (validate) {
+      boolean hasDiff = false;
+
+      for (String category : CATEGORIES) {
```

Comment:
can this part be refactored and not copied; looks similar to the validate section for the configuration docs